server/src/main/java/io/unitycatalog/server/service/CatalogService.java

```diff
@@ -53,7 +53,7 @@ public HttpResponse createCatalog(CreateCatalog createCatalog) {
   }
 
   @Get("")
-  @AuthorizeExpression("#defer")
+  @AuthorizeExpression
```

Comment:
not sure what this change is about. It doesn't really matter either way but using `#defer` helps identify that the authorization is intentionally done elsewhere/later.

Reply:
Okey, I'll revert this change. I think it was IDEA to propose it as `#defer` is the default value.